.github/workflows/linux.yml

```diff
@@ -127,6 +141,13 @@ jobs:
           ln -sf /usr/share/zoneinfo/UTC /etc/localtime
           echo UTC > /etc/timezone
 
+      - name: Setup CMake
```

Comment:
Unused step?

Reply:
Do I not need this to install CMake?

Reply:
Don't think so... You can install it directly using `apt`, which is preferred, IMO.

You can probably just add `cmake` to the `- name: Install other dependencies in docker container` step.

Please add it at the beginning of the list (just to keep the package names sorted alphabetically :D), and add a small comment to like I did for the other packages. Thank you.

Reply:
Although you'll probably need to move the timezone configuration at the top of this step, since `cmake` will probably pull a bunch of other packages that require the timezone to be configured. Take these lines:

```
# Configure the timezone, otherwise 'install-qt-action' gets stuck at 'configuring tzdata'.
ln -sf /usr/share/zoneinfo/UTC /etc/localtime
echo UTC > /etc/timezone
```

And move them at the very top of the step.